routes/users.js

```diff
@@ -14,46 +15,46 @@ const saltRounds = 10;
 
 router.get('/', (req, res, next) => {
   User.find({})
-    .then((records) => {
-      res.send(records);
+    .then((users) => {
+      res.send(users);
     })
     .catch(next);
 });
 
 router.post('/', validator(UserBodySchema), (req, res, next) => {
   User.create(req.body)
-    .then((added) => {
-      let body = _.pick(added, ['_id', 'firstName', 'lastName', 'email'])
-      res.send(body)
+    .then((user) => {
+      const body = _.pick(user, ['_id', 'firstName', 'lastName', 'email']);
+      res.send(body);
     })
-    .catch(next)
-})
+    .catch(next);
+});
 
 router.get('/:usrId', (req, res, next) => {
   User.findById(req.params.usrId)
-    .then((record) => {
-      res.send(record);
+    .then((user) => {
+      res.send(user);
     })
-    .catch(next)
-})
+    .catch(next);
+});
 
 router.delete('/:usrId', (req, res, next) => {
   User.findByIdAndRemove(req.params.usrId)
     .then(() => {
       res.sendStatus(204);
     })
-    .catch(next)
-})
+    .catch(next);
+});
 
 router.put('/:usrId', validator(UserBodySchema), (req, res, next) => {
   if (req.body.password) {
```

Comment:
Remember that in a future you will have to check if a user is editing it's own profile - not someone's others.